documentation/chatmeleon-swagger.yaml

```diff
@@ -337,6 +540,47 @@ components:
             $ref: "#/components/schemas/ObjectId"
             description: List of message IDs the user has seen
 
+    Account:
```

Comment:
Do we ever need to use this schema definition in any client API definition (in request, response)?

Reply:
yea we dont need it so i guess we can remove it?